bin/hyperspy

```diff
@@ -145,7 +145,13 @@ for f in to_copy:
             destination, os.path.split(f)[-1])) or \
             args.overwrite_profile:
         shutil.copy(f, destination)
-
```

Comment:
I think that the `startup` folder feature was introduced in IPython 1.0, couldn't we then use this nicer approach with IPython >= 1.0?